packages/default/scss/button/_theme.scss

```diff
@@ -170,44 +164,98 @@
         // Hover state
         &:hover,
         &.k-state-hover {
-            border-color: $button-hovered-text;
-            color: $button-hovered-bg;
-            background: $button-hovered-text;
-        }
-
-        // Pressed state
-        &:active,
-        &.k-state-active {
-            border-color: $button-pressed-text;
-            color: $button-pressed-bg;
-            background: $button-pressed-text;
+            @include fill( contrast-wcag( $button-text ), $button-text, $button-text, none );
         }
 
         // Focused state
         &:focus,
         &.k-state-focused {
             box-shadow: $button-focused-shadow;
         }
+
+        // Active state
+        &:active,
+        &.k-state-active {
+            @include fill( contrast-wcag( $button-text ), $button-text, $button-text, none );
```

Comment:
This previously used the `$button-pressed-*` variables. Is the change intentional?

Reply:
The outline button depends solely on text color and nothing more, hence the change.